lib/bugsnag/resque.rb

```diff
@@ -37,7 +37,7 @@ def save
 # Auto-load the failure backend
 Bugsnag::Resque.add_failure_backend
 
-Resque.before_first_fork do
+Resque.after_fork do
   Bugsnag.configuration.app_type = "resque"
   Bugsnag.configuration.delivery_method = :synchronous
```

Comment:
Well that doesnt sound good, we should try to get a solution for this. Its tricky because detection around if we are running in a resque app happens later than the configure block is called - so we will have to make sure we get the precedence right here of those options.